pandas/core/indexes/datetimes.py

```diff
@@ -500,6 +500,12 @@ def to_series(self, keep_tz=False, index=None, name=None):
 
               Series will have a datetime64[ns] dtype. TZ aware
               objects will have the tz removed.
+
+            .. versionchanged:: 0.24
+                The default value will change to True in a future release.
```

Comment:
I would just remove the keyword in the future, we don't need to offer this additional funtionaility, this was mainly for backwards compat originally.

Reply:
But, we can of course already discourage the 'False' behaviour, by also deprecating that (only allowing `keep_tz=True`

Reply:
yeah I think i would deprecate any non-None